multiworld/core/gym_to_multi_env.py

```diff
@@ -0,0 +1,118 @@
+import random
+
+import cv2
+import mujoco_py
+import numpy as np
+import warnings
+from PIL import Image
+from gym.spaces import Box, Dict
+
+from multiworld.core.multitask_env import MultitaskEnv
+from multiworld.core.wrapper_env import ProxyEnv
+from multiworld.envs.env_util import concatenate_box_spaces
+from multiworld.envs.env_util import get_stat_in_paths, create_stats_ordered_dict
+
+from gym.spaces import Box, Dict
+
+class GymToMultiEnv(ProxyEnv): # MultitaskEnv):
```

Comment:
This seems more like `MujocoGymToMultiEnv`, right? Could be worth separating out the mujoco-specific code since I could imagine having just a general `GymToMultiEnv` could be useful

Reply:
OK I'll refactor this to two classes